File: MODUL 1/PRAK110-2610817110006-AbdulKhair.c
#include <stdio.h>
#include <math.h>

int main() {
    float alas = 5;
    float tinggi = 12;
    float hypotenuse;
    float perimeter;
    float area;

    hypotenuse = sqrt((alas * alas) + (tinggi * tinggi));
    perimeter = alas + tinggi + hypotenuse;
    area = 0.5 * alas * tinggi;

    printf("Diketahui :\n");
    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", tinggi);
    printf("\nJawab :\n");
    printf("Sisi A = %0.f cm\n", tinggi);
    printf("Sisi B = %0.f cm\n", hypotenuse);
    printf("Sisi C = %.0f cm\n", alas);
    printf("Keliling = %.0f cm\n", perimeter);
    printf("Luas = %.0f cm\n", area);
    return 0;
}

File: MODUL 1/Bahasa Python/PRAK110-2610817110006-AbdulKhair.py
import math

alas = 5
tinggi = 12
hypotenuse = math.sqrt((alas ** 2) + (tinggi ** 2))
perimeter = alas + tinggi + hypotenuse
area = 0.5 * alas * tinggi

print("Diketahui :")
print(f"alas = {alas} cm")
print(f"tinggi = {tinggi} cm")
print("\nJawab :")
print(f"Sisi A = {tinggi} cm")
print(f"Sisi B = {hypotenuse:.0f} cm")
print(f"Sisi C = {alas} cm")
print(f"Keliling = {perimeter:.0f} cm")
print(f"Luas = {area:.0f} cm")